src/test/java/uk/ac/ebi/ena/webin/cli/service/SampleServiceTest.java

```diff
@@ -54,6 +54,24 @@ public void testGetSampleUsingInvalidId() {
         sampleService.getSample( id );
     }
 
+    @Test
+    public void testPreNov22PrivateSampleRetrieval() {
+        // This is a private Pre Nov 2022 sample whose authority is ENA not Biosamples.

```

Comment:
// Test sample retrieval of a private ENA sample submitted using Webin before November 2022.

// Private samples submitted to ENA using Webin before November 2022 are not available 
// from BioSample because these samples will only be added to BioSample once they become 
// public. ALL samples submitted to Webin after November 2022 are immediately added to both 
// ENA and BioSamples.